### PR TITLE
Fix init message for FMI

### PR DESCRIPTION
**Pull request by ptaeuber**

### CODE REVIEW

File: SimulationRuntime/c/simulation/solver/solver_main.c [hunk 1]
```diff
@@ -465,6 +465,8 @@ int initializeModel(DATA* data, threadData_t *threadData, const char* init_initM
       simInfo->stopTime = simInfo->startTime;
       retValue = -1;
     }
+    if (!retValue)
+      infoStreamPrint(LOG_STDOUT, 0, "The initialization finished successfully %s homotopy method.", data->simulationInfo->homotopyUsed ? "with" : "without");
```
Comment (sjoelund) [inline SimulationRuntime/c/simulation/solver/solver_main.c]: Don't log this to STDOUT; if you do we can't disable it in the logging settings

Comment (lochel) [reply SimulationRuntime/c/simulation/solver/solver_main.c]: We can add an option to disable `LOG_STDOUT` or we can have a new default notification stream that is enabled by default – which is then basically the same as having an option to disable `LOG_STDOUT`.

Comment (ptaeuber) [reply SimulationRuntime/c/simulation/solver/solver_main.c]: I also thought about these two possibilities. We could have the option `-lv=NO_STDOUT`.

Comment (sjoelund) [reply SimulationRuntime/c/simulation/solver/solver_main.c]: Don't touch stdout. This belongs to LOG_INIT or a separate category.

**Comment by ptaeuber** [comment]
@sjoelund @lochel Now the defaultly printed success messages are moved from stdout to a new stream LOG_SUCCESS, which can be deactivated by `-lv=-LOG_SUCCESS`.

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2017-11-06_16-44-46](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4881/).

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2017-11-06_17-36-18](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/4883/console).

**Pull request closed without merging**